### PR TITLE
Redirect asset downloads

### PR DESCRIPTION
**Pull request by nicpottier**
This moves to naming files with UUIDs for exports as well as using 301 redirects for downloads so we don't time out trying to download large files before serving to nginx.

### CODE REVIEW

File: temba/assets/models.py [hunk 1]
```diff
@@ -47,56 +46,71 @@ class BaseAssetStore(object):
     def __init__(self, asset_type):
         self.asset_type = asset_type
 
-    def resolve(self, user, identifier):
+    def resolve(self, user, pk):
```
Comment (nicpottier) [inline temba/assets/models.py]: Renamed all uses of 'identifier' to 'pk' as it confused me. (i thought it could be any identifier (like a UUID) not a pk)

**Comment by nicpottier** [comment]
Note that this WILL break old export links, but I'm ok with that.

**Comment by nicpottier** [comment]
Ok, ready for ze review. This has been pretty well tested on staging

**Comment by ericnewcomer** [comment]
This all seems reasonably sane and seems you've covered the places I can think of to replace. Did you try playing flow recordings in the UI by chance?

File: temba/assets/models.py (hunk 1 above)
Comment (rowanseymour) [inline temba/assets/models.py]: I think that was the original plan

**Comment by rowanseymour** [comment]
Yarp

**Comment by nicpottier** [comment]
Ya, tested this on staging playing both uploaded recordings and recordings from flows.

**Pull request merged**